meshtastic/interfaces/ble/reconnection.py

```
@@ -154,6 +228,24 @@ def _schedule_reconnect(self, auto_reconnect: bool, shutdown_event: Event) -> bo
             raise
         return True
 
+    def schedule_reconnect(self, auto_reconnect: bool, shutdown_event: Event) -> bool:
```

Ruff (0.15.5)

[warning] Boolean-typed positional argument in function definition

(FBT001)